src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,17 @@
-@mixin hover($_property, $_toValue) {
```

Comment:
The hover mixin has been removed. It is a very useful mixin for handling hover states and it should not be removed unless it is replaced with a better solution.